Core/Src/main.c

```diff
@@ -361,14 +314,24 @@ int main(void)
         osThreadCreate(osThread(transmitDataThread), allData);
 
     osThreadDef(
-        abortPhaseThread,
-        abortPhaseTask,
+        flightPhaseThread,
+		flightPhaseTask,
+        osPriorityHigh,
+        1,
+        configMINIMAL_STACK_SIZE * 3
+    );
+    logDataTaskHandle =
```

Comment:
should this be assigned to the logDataTaskHandle?

Reply:
changed to flightPhaseTaskHandle.